sbc_compassion/tools/patternrecognition.py

```diff
@@ -120,7 +120,7 @@ def findMatches(des1, des2, test=0.8):
     :rtype: list[Keypoints]
     """
     bf = cv2.BFMatcher()
-    matches = bf.knnMatch(des2, des1, k=2)
+    matches = bf.knnMatch(des1, des2, k=2)
```

Comment:
I had some trouble with the images for the test and usually this order is given (and seems more logical). I do not know exactly the difference, but I think I have done it wrong before this modification